src/components/basicComponents/SortByCategoryItem.vue

```diff
@@ -1,13 +1,25 @@
 <template>
-  <li>
-    <BaseButton>{{ text }}</BaseButton>
+  <li v-for="category in UNIQUE_CATEGORIES" :key="category">
+    <BaseButton
+      :text="category"
+      :value="category"
+      @click="filteredData(category)"
+    />
   </li>
 </template>
 
 <script setup>
 import BaseButton from "./BaseButton.vue";
+import { computed, ref } from "vue";
+import { useFeedbacksStore } from "@/stores/feedbacks.js";
+import { useUserStore } from "@/stores/user.js";
 
-defineProps({
-  text: { type: String, required: true },
-});
+const usersStore = useUserStore();
+const feedbacksStore = useFeedbacksStore();
+const UNIQUE_CATEGORIES = computed(() => feedbacksStore.UNIQUE_CATEGORIES);
```

Comment:
it depends (as always), but I'm used to using lower case in components and upper case in stores or in variables f.e const uniqueCategories = ...

Reply:
ok, I'll note this comment and I will change the naming convention after our meeting on Monday because I am not sure if I understand well your comment.